avocado/utils/nvme.py

```
@@ -529,3 +529,41 @@ def get_subsystem_using_ctrl_name(ctrl):
+
+def get_nvme_subsystem_io_policy(subsystem):
+    """
+    Read io policy parameter from sysfs folder and returns a string
+
+    :param subsystem: Name of the subsystem
+    :rtype: String
+    """
+    subsys_iopolicy_path = f"/sys/class/nvme-subsystem/{subsystem}/iopolicy"
+    if os.path.isfile(subsys_iopolicy_path):
+        return open(subsys_iopolicy_path, "r", encoding="utf-8").readline().rstrip("\n")
+    raise NvmeException(f"iopolicy file not found: {subsys_iopolicy_path}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)